components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -1384,11 +1369,11 @@ void RRTMGPRadiation::run_impl (const double dt) {
       );
 #endif
 #ifdef RRTMGP_ENABLE_KOKKOS
-      rrtmgp::rrtmgp_main(
+      interface_t::rrtmgp_main(
         ncol, m_nlay,
-        p_lay_k, t_lay_k, p_lev_k, t_lev_k,
+        p_lay_k, t_lay_k, p_lev_k, d_tint,
         m_gas_concs_k,
-        sfc_alb_dir_k, sfc_alb_dif_k, mu0_k,
+        sfc_alb_dir_k, sfc_alb_dif_k, d_mu0,
```

Comment:
I'm a little confused about use of `d_mu0` here. It looks like `d_mu0` refers to the m_buffer field cosine_zenith (`auto d_mu0 = m_buffer.cosine_zenith;`), but I don't see it populated anywhere above. Should this be another subview slice for this chunk?

Reply:
Okay never mind, I think I found it. But I think maybe use of `d_mu0` for naming is a little confusing here. I had originally taken the `d_` prefix convention from some other process (don't remember which) to refer to the device views that come directly from the FM. In this case though, I think `d_mu0` is being treated like the other subviews that hold data for just the current "chunk" of ncol. Should we rename this to be consistent with how the others are used? Or am I missing something here?

Reply:
Any name is fine with me, I was just using the names that were already there. mu0_k was just a copy of d_mu0.